web/libs/datamanager/src/utils/feature-flags.js

```diff
@@ -55,6 +55,11 @@ export const FF_MEMORY_LEAK_FIX = "fflag_feat_all_optic_1178_reduce_memory_leak_
  */
 export const FF_REGION_VISIBILITY_FROM_URL = "fflag_feat_front_optic_1553_url_based_region_visibility_short";
 
+/**
+ * Add ability to show average agreement score popover in Agreement cell.
+ */
+export const FF_AVERAGE_AGREEMENT_SCORE_POPOVER = "fflag_feat_all_leap_2042_average_agreement_score_popover";
```

Comment:
This was my mistake, thanks for spotting it. I was amending it in this [PR](https://github.com/HumanSignal/label-studio/pull/7582) but happy to see it fixed here. Only thing though, we need to remove it from the `editor` feature-flags.ts

Reply:
no worries, if that pr is ready (i just approved it) lets get it merged and i'll merge the changes back in here